### PR TITLE
Disable PR and commit testing for MoveIt 2

### PR DESCRIPTION
**Pull request by gavanderhoorn**
These jobs keep failing / timing out, and we already have CI based on GHA over at `ros-planning/moveit2` (targetting the different active ROS 2 versions, including Rolling).

As MoveIt builds can take a *really* long time (without the extensive caching we do at `ros-planning/moveit2`), this PR suggests to save resources by removing the `source` entries for Foxy, Galactic and Rolling respectively, so the buildfarm can do something else.

### CODE REVIEW

**Review by jacobperron — APPROVED** [review]
Fine by me. I'd like a second opinion.

**Comment by mikeferguson** [comment]
Would it not be better to set:

```
      test_commits: false
      test_pull_requests: false
```

For both entries - that way the source can still be found by automated tools

**Comment by gavanderhoorn** [comment]
Good suggestion @mikeferguson.

Updated.

**Review by audrow — APPROVED** [review]

**Review by jacobperron — APPROVED** [review]
LGTM with a +1 from a moveit2 maintainer, maybe @henningkayser?

**Review by henningkayser — APPROVED** [review]
Good idea to just disable the build jobs explicitly instead of removing the entries altogether. LGTM

**Pull request merged**
